content/en/dashboards/ddsql_editor/reference/data_types.md

```diff
@@ -8,16 +8,15 @@ DDSQL is in private beta.
 
 ## Data types
 
-DDSQL implements a simplified version of the SQL type system that is mostly descended from SQLite.
+DDSQL implements a simplified version of the SQL type system that is mostly descended from PostgreSQL.
```

Comment:
While it may bear similarities to the SQLite type system, SQLite is a dynamically typed SQL, and DDSQL is statically typed. Both historically and currently, DDSQL adheres to PostgreSQL where possible.